gammapy/modeling/models/spatial.py

```diff
@@ -612,6 +647,55 @@ def evaluation_region(self):
         """Evaluation region consistent with evaluation radius"""
         return self.to_region(x_sigma=5)
 
+    def plot_extension_error(self, ax=None, x_sigma=1.5, **kwargs):
+        r"""Plot model error at a given number of :math:`\sigma`.
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`, optional
+                Axis
+        x_sigma : float
+            Number of :math:`\sigma
+            Default is :math:`1.5\sigma` which corresponds to about 68%
+            containment for a 2D symmetric Gaussian.
```

Comment:
This is just the error on the extension that is plotted, no? Plotting the 1-sigma error would be more standard I think